Author: Andreas Tille <tille@debian.org>
Last-Update: Fri, 17 Aug 2018 13:58:34 +0200
Description: bif_string_arg does not return int - fix type

--- a/libsrc/Wi/bif_crypto.c
+++ b/libsrc/Wi/bif_crypto.c
@@ -1816,7 +1816,7 @@ bif_base36enc (caddr_t * qst, caddr_t *
 static caddr_t
 bif_base36dec (caddr_t * qst, caddr_t * err_ret, state_slot_t ** args)
 {
-  boxint str = bif_string_arg (qst, args, 0, "base36dec");
+  caddr_t str = bif_string_arg (qst, args, 0, "base36dec");
   boxint ret;
   ret = strtoul(str, NULL, 36);
   return box_num (ret);
